source/common/stats/symbol_table_impl.cc

```diff
@@ -442,19 +442,35 @@ void StatNameSet::rememberBuiltin(absl::string_view str) {
   builtin_stat_names_[str] = stat_name;
 }
 
-Stats::StatName StatNameSet::getStatName(absl::string_view token) {
+StatName StatNameSet::getBuiltin(absl::string_view token, StatName fallback) {
+  StatName stat_name = getBuiltinHelper(token, fallback);
+  if (stat_name == fallback) {
+    // TODO(#8116): do periodic logging here.
+    // ENVOY_LOG_MISC(warn, "Builtin stat name not pre-registered: {}", token);
```

Comment:
I'm still skeptical that the logging is the best path forward. What about doing something like:
1) Increment some stat when we hit this path.
2) Possible register the name with some admin handler, which could then be dumped by an operator to debug?

Reply:
Sorry I realize you are doing something like this in the other PR. I will review that today and then we can circle back to this call-site.

Reply:
I removed the commented-out code and the helper function, per the discussion on Slack where you convinced me not to add logging initially.